locals/itIT.lua

```diff
@@ -0,0 +1,9 @@
+-- itIT localization
+local L = LibStub("AceLocale-3.0"):NewLocale("WOWUP", "itIT")
+
+L["Notification"] = Notifica
+L["You have %d addon to be updated"] = Hai %d addon da aggiornare
```

Comment:
Can you add the `"` around the sentences? Otherwise it's not valid lua. Example:

```diff
- Hai %d addon da aggiornare
+ "Hai %d addon da aggiornare"
```

Reply:
Oh sorry. I've added them, hope it is correct now.